consensus/obcpbft/pbft-core.go

```
@@ -279,7 +281,29 @@ func (instance *pbftCore) processEvent(e interface{}) interface{} {
 	case pbftMessageEvent:
 		msg := et
 		logger.Debug("Replica %d received incoming message from %v", instance.id, msg.sender)
-		instance.recvMsg(msg.msg, msg.sender)
+		next, err := instance.recvMsg(msg.msg, msg.sender)
+		if err != nil {
```

Is there ever a case where we don't want to return a nil error?